### DISCUSSION TITLE
Move a specified workspace to a different output

### DISCUSSION THREAD

**Discussion by j8s0n** [i3/i3#4933] [Q&A]
I'm trying to write a script to run when I connect and disconnect my notebook to an external display. When I disconnect the display, any workspaces on that display disappear. I found the command `i3-msg move workspace to output left`, which does what I want, but I have to be on that workspace. Is there a way to script telling it to move a workspace to another display?

**Comment by NekoiNemo** [comment]
There isn't one to do that, but you can just chain commands like `$ i3-msg 'workspace 4; move workspace to output right'`. With an obvious downside of it now being the focused one, ~~and there being no easy way to go back to the original one, other than the `workspace back_and_forth`~~, nevermind, that focuses the output that is now not available, not the previous active workspace

**Reply by Airblader** [reply]
Small note: you're going to want to use `--no-auto-back-and-forth` for such scenarios.
